conda_recipes/aftereffects-25.0/README.md

```diff
@@ -0,0 +1,46 @@
+# Adobe After Effects 25 conda build recipe
+
+## Creating an archive file for Windows
+
+The Windows installer requires Administrator permissions that are not available in most conda package
+build environments, such as on a Deadline Cloud service-managed fleets. Follow these instructions to
+install Adobe After Effects 25 on a freshly created EC2 instance as Administrator, and create an archive file
+for use by the conda build recipe.
```

Comment:
Should a note be added that this is one way to create the archive? This could very easily also be created on any Windows machine they may have physically no?

Reply:
Will update.